aws-ecs/telegraf.conf

```diff
@@ -54,8 +54,8 @@
 
   ## ecs labels to include and exclude as tags.  Globs accepted.
   ## Note that an empty array for both will include all labels as tags
-  ecs_label_include = [ "com.amazonaws.ecs.*" ]
-  ecs_label_exclude = []
+  ecs_label_include = []
+  ecs_label_exclude = [ "com.amazonaws.ecs.*" ]
```

Comment:
This change excludes the point tags with prefix - "com.amazonaws.ecs.*" avoiding the duplicates